src/webapp/src/commands/index.js

```diff
@@ -120,6 +120,7 @@ const commands = {
     _package: 'volume',
     plugin: 'ctrl',
     method: 'set_volume',
+    argKeys: ['volume'],
```

Comment:
Unrelated to this change?

Reply:
yes. didn't want to create a separate PR for it